README.md

```diff
@@ -29,6 +29,10 @@ Once an enjoyable one is found:
 - `nvim ~/.config/nvim/private.vim`
 - Look for `colorscheme jellybeans` and replace it with the new one.
 
+_Optionnal_:
+
+- For design of Infra as code, you could add HCL linter by runnig `go get github.com/fatih/hclfmt`
```

Comment:
```
" Plugin outside ~/.vim/plugged with post-update hook
Plug 'junegunn/fzf', { 'dir': '~/.fzf', 'do': './install --all' }
``` 

It's something like that according to [Plug's](https://github.com/junegunn/vim-plug) README